app/javascript/profileDropdown/flagButton.js

```diff
@@ -1,32 +1,12 @@
 /* global userData */
-/* eslint-disable no-alert */
-/**
- * Adds a flag button visible only to trusted users on profile pages.
- * @function initFlag
- * @returns {(void|undefined)} This function has no useable return value.
- */
-export function initFlag() {
-  const flagButton = document.getElementById(
-    'user-profile-dropdownmenu-flag-button',
-  );
-
-  if (!flagButton) {
-    // button not always present when this is called
-    return;
-  }
-
-  const user = userData();
-  if (!user) {
-    return;
-  }
+/* eslint-disable no-alert, import/order */
+import { request } from '@utilities/http';
+import { getUserDataAndCsrfToken } from '../chat/util';
 
+function addFlagUserBehavior(flagButton) {
```

Comment:
names are hard - what is this "install event handlers on an element" function called?